contents/newsletter/choosing-technologies.md

```diff
@@ -0,0 +1,154 @@
+
+- **Reliability:** Companies rely on us to handle business-critical data. We need to maintain a high uptime and availability. Technologies we choose need to prove they are reliable and handle failures gracefully.
+
+- **Support:** Is this a shiny new thing or a battle-tested one? Is it open source with an active community? We know problems will arise, so we need to know we can get help when we need it. For all of our most important technologies, we have experts we can talk to if we need help.
+
+- **Flexibility:** Because we plan to ship many products beyond what we have today, we need technologies that are interoperable and can be easily adapted to new use cases.
+
+You'll notice that some of these are business factors and others are technical. Where some software engineers would purely focus on the technical factors, our team of [product engineers](/blog/what-is-a-product-engineer) know they need to consider both.
+
+## Make a decision asynchronously
+
+Although individuals are empowered to test and evaluate new technologies, for decisions affecting multiple teams, the final decision is made through a **request for comments** (RFC). These help us get buy-in, gather feedback on a decision, and stay transparent. The process of writing them all forces our team to clearly articulate their thoughts in a structured way.
+
+An RFC outlines the "why" behind a technology decision and helps facilitate input. The specific parts of an vary, but broadly include the following parts:
+
+- What is the problem you are trying to solve?
+- What are the options you considered?
+- Based on your evaluation, which one do you recommend?
+
+This isn't just a box to tick, it is a core part of helping us choose the right technologies. For example, our RFC for adopting [Temporal at PostHog](https://github.com/PostHog/meta/pull/99) had 2,594 words, roughly 1/4 of the company as reviewers, 46 comments, and quite a bit of disagreement before finally being adopted. It's now happily in use for [batch exports](/blog/temporal-exports) and [warehouse syncs](/docs/data-warehouse/setup).
+
+Whatever happens, the final decision usually rests on the team implementing and maintaining the technology. Even if an agreement isn't reached, if they feel confident it is the right solution, they have leeway to go ahead with adoption.
+
+> **Related:** [How we work asynchronously](https://newsletter.posthog.com/p/how-we-work-asynchronously)
+
+## When do we say NO to technologies?
+
+If someone goes through all of this work, the expected answer for adopting a technology is yes. What has happened in when that wasn't the case?
+
+- **Different perspectives on the evaluation criteria.** For example, the benefit of adopting a new technology for another team is not as valuable as expected, ruining the overall benefit.
```

Comment:
this is a bit hard to parse atm. are you talking about a change having a negative / net-neutral impact on another team, rather than the one proposing it? It might be useful to provide a real or hypothetical example here.

Reply:
rewrote